static/data/colors.json

```diff
@@ -30,6 +30,8 @@
     "#4c96cb",
     "#142E58"
   ],
+  "Geo": "#879bf5",
+  "NAPCS": "#EEA86C",
```

Comment:
Instead of putting these here, can we just use the `style.yml` colors? (there are already variables for each profile slug, `geo`, `naics`, etc)